modules/govuk_ci/manifests/agent.pp

```diff
@@ -54,4 +54,10 @@
   package { 'libgdal-dev': # needed for mapit
     ensure => installed,
   }
+
+  cron::crondotdee { 'clean_up_bundles':
+    command => 'find /var/lib/jenkins/bundles/* -maxdepth 0 -type d -mtime +1 -exec rm -rf {} \;',
```

Comment:
Strange that you're getting different results.
```
root@ci-agent-1:~# find /var/lib/jenkins/bundles/* -maxdepth 0 -type d -mtime +1
/var/lib/jenkins/bundles/aa-test
/var/lib/jenkins/bundles/businesssupportfinder
/var/lib/jenkins/bundles/calculators
/var/lib/jenkins/bundles/calendars
/var/lib/jenkins/bundles/collections
/var/lib/jenkins/bundles/collections-publisher
/var/lib/jenkins/bundles/contacts
/var/lib/jenkins/bundles/contacts-frontend
/var/lib/jenkins/bundles/content-performance-manager
/var/lib/jenkins/bundles/content-tagger
/var/lib/jenkins/bundles/email-alert-frontend
/var/lib/jenkins/bundles/email-alert-service
continued..........
```
Perhaps you require a full login???

Maybe same command with modified argument `-maxdepth 1`??
```
root@ci-agent-1:~# find /var/lib/jenkins/bundles/* -maxdepth 1 -type d -mtime +1
/var/lib/jenkins/bundles/aa-test
/var/lib/jenkins/bundles/aa-test/deployed-to-production
/var/lib/jenkins/bundles/aa-test/deployed-to-staging
/var/lib/jenkins/bundles/businesssupportfinder
/var/lib/jenkins/bundles/businesssupportfinder/deployed-to-production
/var/lib/jenkins/bundles/calculators
/var/lib/jenkins/bundles/calculators/master
/var/lib/jenkins/bundles/calendars
/var/lib/jenkins/bundles/calendars/move-to-jenkins-2
/var/lib/jenkins/bundles/collections
/var/lib/jenkins/bundles/collections/nav-redirects
/var/lib/jenkins/bundles/collections/remove-parent-link
/var/lib/jenkins/bundles/collections/add-vary-header
/var/lib/jenkins/bundles/collections/remove-nth-selectors-from-grid
continued...........
```